test/functionalTest/cases/4113_alarm_for_notification_queue_overpassing_a_given_threshold/alarm_for_notification_queue_overpassing_a_given_threshold.test

```diff
@@ -0,0 +1,230 @@
+# Copyright 2023 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+alarm for notification queue overpassing a given threshold
+
+--SHELL-INIT--
+dbInit ${CB_DB_NAME} serv1
+dbInit ${CB_DB_NAME} serv2
+dbInit ${CB_DB_NAME} serv3
+brokerStart CB 0 IPv4 -multiservice -notificationMode threadpool:3:2
```

Comment:
You should define per-service queues, something like this:

```
-notificationMode threadpool:3:2,serv1:3:2,serv2:3:2,serv3:3:2
```

Please have a look to case/3843 tests for inspiration,

Reply:
Updated in https://github.com/telefonicaid/fiware-orion/pull/4375/commits/51a71bbd8113d91b0c1858d8f1f1aada294a61be